src/routes/github/manifest/github-manifest-get.ts

```diff
@@ -28,7 +28,7 @@ const getAppManifest = () => {
 		"default_permissions": {
 			"actions": "read",
 			"security_events": "read",
-			"contents": "read",
+			"contents": "write",
 			"deployments": "read",
```

Comment:
@joshkay10, we need to update app permission for creating branches.  I believe, this is well documented and published as public docs.  So we've update those docs.